inc/based_config.php

```diff
@@ -57,6 +52,11 @@
    define("GLPI_CONFIG_DIR", GLPI_ROOT . "/config");
 }
 
+// If this file exists, it is load
+if (file_exists(GLPI_CONFIG_DIR. '/local_define.php')) {
```

Comment:
Miss second part of original condition

Reply:
You are right, I missed it as it's not included in the release I installed 9.3, but a git blame showed me that you added this condition, so if it resolve the problem, thats fine for me.